### PR TITLE
python 3 migration: long

### PR DESCRIPTION
**Pull request by philipwinklerdelmic**

### CODE REVIEW

File: src/odemis/driver/simcam.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def __init__(self, name, role, image, dependencies=None, daemon=None, blur_facto
                     (hlf_shape[0], hlf_shape[1])]
         self._translation = (0, 0)
         self.translation = model.ResolutionVA(self._translation, tran_rng,
-                                              cls=(int, long), unit="px",
+                                              cls=(int), unit="px",
```
Comment (pieleric) [inline src/odemis/driver/simcam.py]: You can drop `cls=...` entirely, as it's the default of `ResolutionVA`.

File: src/odemis/driver/andorcam3.py [hunk 2]
```diff
@@ -331,7 +331,7 @@ def __init__(self, name, role, device=None, bitflow_install_dirs=None, **kwargs)
             tran_rng = ((-uh_shape[0], -uh_shape[1]),
                         (uh_shape[0], uh_shape[1]))
             self.translation = model.ResolutionVA((0, 0), tran_rng,
-                                                  cls=(int, long), unit="px",
```
Comment (pieleric) [inline src/odemis/driver/andorcam3.py]: Drop `cls=...` entirely.

File: src/odemis/acq/stream/_live.py [hunk 3]
```diff
@@ -1023,15 +1023,15 @@ def __init__(self, name, detector, dataflow, emitter, **kwargs):
         # Resolution assumes that we scan the whole roi. For smaller ROIs, the
         # actual hardware setting is proportional.
         self.resolution = model.TupleContinuous(hwres.value, range=hwres.range,
-                                                cls=(int, long, float),
+                                                cls=(int, float),
```
Comment (pieleric) [inline src/odemis/acq/stream/_live.py]: Actually, `float` doesn't make sense for this VA, so just changing to `cls=int` would be even better.

**Comment by philipwinklerdelmic** [comment]
You're right about the imports. I thought it wasn't necessary in all places, but I thought again and came to the conclusion that it is.

**Comment by philipwinklerdelmic** [comment]
done

File: src/odemis/driver/simcam.py [hunk 4]
```diff
@@ -22,6 +22,7 @@
 from __future__ import division
 
 import queue
+from builtins import int
```
Comment (pieleric) [inline src/odemis/driver/simcam.py]: Unused?

File: src/odemis/acq/stream/_sync.py [hunk 5]
```diff
@@ -26,6 +26,7 @@
 
 from __future__ import division
 
+from builtins import int
```
Comment (pieleric) [inline src/odemis/acq/stream/_sync.py]: Unsused

File: src/odemis/driver/andorcam3.py [hunk 6]
```diff
@@ -28,6 +28,7 @@
 
 from __future__ import division
 
+from builtins import int
```
Comment (pieleric) [inline src/odemis/driver/andorcam3.py]: Unused?

**Comment by philipwinklerdelmic** [comment]
done

**Pull request merged**